### PR TITLE
feat(SQS): app wiring; configs; makefile; localosmosis

### PR DESCRIPTION
**Pull request by p0mvn**
Closes: #XXX

## What is the purpose of the change

This PR includes the following components necessary for getting the SQS MVP in:
- SQS app wiring
- Picking up configs from `app.toml`
  * Disable by default
  * Other parameters default to what we have in the stage deployment right now
- Makefile commands to interact with SQS
- Some localosmosis changes to enable SQS by default

Additionally, found bug in the healthcheck. Addressed it here as a drive-by. Tested on localosmosis.

Taken from POC: https://github.com/osmosis-labs/osmosis/pull/6785

## Testing and Verifying

- This is what we have our stage deploying to
- Tested on localosmosis

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes?
  - [ ] Changelog entry added to `Unreleased` section of `CHANGELOG.md`?

Where is the change documented? 
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [ ] Code comments?
  - [ ] N/A

### CODE REVIEW

File: tests/localosmosis/scripts/setup.sh [hunk 1]
```diff
@@ -180,60 +206,29 @@ then
...
-        fi
-
-        echo "Generating Osmosis genesis for the concentrated liquidity module from Uniswap data"
-        ./script --operation 1 --localosmosis
-    fi
-
-    # Run genesis merge script
-    ./script --operation 2 --localosmosis
-    cd ..
-fi
```
Comment (p0mvn) [inline tests/localosmosis/scripts/setup.sh:229]: Note: we don't need this anymore

File: tests/localosmosis/scripts/setup.sh [hunk 2]
```diff
@@ -180,60 +206,29 @@ then
...
+    
+    echo "uusdc / uion balancer"
+    create_two_asset_pool "uusdcUionBalancerPool.json"
+
+    echo "stake / uion / uosmo balancer"
     create_three_asset_pool
+
+    echo "uion / uosmo concentrated"
+    create_concentrated_pool
+    create_concentrated_pool_positions
```
Comment (p0mvn) [inline tests/localosmosis/scripts/setup.sh:232]: Note: all of these are broken after the unfork. However, all changes are tested on the v20.x release line.

Made issue to track this: https://github.com/osmosis-labs/osmosis/issues/6979

File: app/app.go [hunk 3]
```diff
@@ -251,6 +255,28 @@ func NewOsmosisApp(
...
+		}
+
+		sqsIngester, err := sqsConfig.Initialize(appCodec, sqsKeepers)
+		if err != nil {
+			panic(err)
+		}
+
+		// Set the sidecar query server ingester to the ingest manager.
+		app.IngestManager.RegisterIngester(sqsIngester)
+	}
```
Comment (p0mvn) [inline app/app.go:278]: Note: it is desabled via config by default

File: cmd/osmosisd/cmd/root.go [hunk 4]
```diff
@@ -461,6 +466,59 @@ min-gas-price-for-high-gas-tx = ".0025"
...
 adaptive-fee-enabled = "true"
+
+###############################################################################
+###              Osmosis Sidecar Query Server Configuration                 ###
+###############################################################################
+
+[osmosis-sqs]
+
+# SQS service is disabled by default.
+is-enabled = "false"
```
Comment (p0mvn) [inline cmd/osmosisd/cmd/root.go:477]: Note: disabled

File: tests/localosmosis/state_export/docker-compose.yml [hunk 5]
```diff
@@ -1,20 +1,38 @@
-version: "4"
+version: "3"
```
Comment (czarcas7ic) [inline tests/localosmosis/state_export/docker-compose.yml:1]: Why did version decrease?

**Review by czarcas7ic — APPROVED** [review]
🧯 🚒 🔥

File: Makefile [hunk 6]
```diff
@@ -164,6 +164,35 @@ go-mock-update:
...
+sqs-profile:
+	go tool pprof -http=:8080 http://localhost:9092/debug/pprof/profile?seconds=15
+
+# Updates go tests with the latest mainnet state
+# Make sure that the node is running locally
+sqs-update-mainnet-state:
+	curl -X POST "http:/localhost:9092/store-state"
+	mv pools.json ingest/sqs/router/usecase/routertesting/parsing/pools.json
+	mv taker_fees.json ingest/sqs/router/usecase/routertesting/parsing/taker_fees.json
+
```
[the remark this answers is not in the document]
Comment (p0mvn) [reply Makefile:195]: Looked into this but made the decision to keep as is. I would need to validate that many relative paths continue to work.

These steps will be removed soon anyways once we move out sqs out-of-process

**Pull request merged**